staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/admission.go

```diff
@@ -245,7 +262,76 @@ func (a *GenericAdmissionWebhook) Admit(attr admission.Attributes) error {
 	return errs[0]
 }
 
+func (a *GenericAdmissionWebhook) getNamespaceLabels(attr admission.Attributes) (map[string]string, error) {
+	// Get the labels from attr.Object if the request itself is creating a
```

Comment:
I would rearrange with as if <> then <>. Also I would collapse the two if comments.

Reply:
Done.